doc/services/zbus/index.rst

```diff
@@ -424,8 +424,11 @@ exchanges accelerometer data, for example.
     K_THREAD_DEFINE(subscriber_task_id, 512, subscriber_task, NULL, NULL, NULL, 3, 0, 0);
 
     ZBUS_MSG_SUBSCRIBER_DEFINE(my_msg_subscriber);
-    static void msg_subscriber_task(void *sub)
+    static void msg_subscriber_task(void *ptr1, void *ptr2, void *ptr3)
```

Comment:
This is not necessary if you are not using the arguments. The current signature is wrong indeed, but for solving that you need to remove the `sub` parameter. 
```suggestion
    static void msg_subscriber_task()
```
That should be enough.

Reply:
The most common pattern in Zephyr that I saw is defined function with arguments even though they are unused. The same is done in the threads docs area, please see:

<pre>
#define MY_STACK_SIZE 500
#define MY_PRIORITY 5

extern void my_entry_point(void *, void *, void *);

K_THREAD_STACK_DEFINE(my_stack_area, MY_STACK_SIZE);
struct k_thread my_thread_data;

k_tid_t my_tid = k_thread_create(&my_thread_data, my_stack_area,
                                 K_THREAD_STACK_SIZEOF(my_stack_area),
                                 my_entry_point,
                                 NULL, NULL, NULL,
                                 MY_PRIORITY, 0, K_NO_WAIT);
</pre>

Reply:
A typical pattern if you take a look at the samples is using only `extern void my_entry_point(void)`. I cannot see any benefit in keeping the parameter there.

Reply:
Alright, I will change it.

Reply:
> A typical pattern if you take a look at the samples is using only `extern void my_entry_point(void)`. I cannot see any benefit in keeping the parameter there.

please see https://github.com/zephyrproject-rtos/zephyr/pull/62903
Should the tasks have the wrong signature and inadvertently return, they will corrupt the stack. I think it's good practice to make sure the signature is correct, even if that causes extra `ARG_UNUSED` stuff...

Reply:
Interesting, so keep that @bbilas

Reply:
I had my finger on the trigger but you were faster ;)

Reply:
Based on that, we have other samples and test code to fix. :sweat_smile: